source/codegen/templates/service_helpers.mako

```diff
@@ -6,7 +6,7 @@
 ## Generate the core method body for an Init method. This should be what gets included within the try block in the service method.
 <%def name="define_init_method_body(function_name, function_data, parameters)">\
 <%
-  config = data['config']
+  config = data['config'] 
```

Comment:
[Minor] Looks like an extra space was added after the closing bracket. Can you remove it?

Reply:
Done. Thanks for putting it to notice.